tribe-ext-link-events-to-website-url/index.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar: Link Event Titles to Event Website URLs
```

Comment:
I think this is a prime example of what we were debating in our Slack DMs today. 

If length is a concern, adding 10-character word (9chars+a space) with dubious value is not the best move we can make here.

We go from 60 chars:
`The Events Calendar: Link Event Titles to Event Website URLs`
To 58 chars:
`The Events Calendar Extension: Link Events to Website URLs`

☝️ And in my opinion the extra clarity in the above option is _worth_ the 2 extra chars. If you disagree about that, I understand. But then let's look at the title with your shortened title _and_ `Extension:` omitted:

`The Events Calendar: Link Events to Website URLs`
☝️ A marvelous 48 characters. 

We'll come back to this later, so for now I will implement the `Extension:` as we discussed. But I think this example illustrates its uselessness very, very well.